hutch_python/load_conf.py

```diff
@@ -222,6 +228,18 @@ def load_conf(conf, hutch_dir=None):
         with safe_load('scan_pvs (disabled)'):
             cache(scan_pvs=ScanVars('{}:SCAN'.format(hutch.upper()),
                                     name='scan_pvs', RE=RE))
+    # Elog
+    if hutch is not None:
+        with safe_load('elog'):
+            # Use the fact if we we used the default_platform or not to decide
+            # whether we are in a specialty station or not
+            if default_platform:
+                logger.debug("Using primary experiment ELog")
+                kwargs = dict()
+            else:
+                logger.info("Configuring ELog to post to secondary experiment")
+                kwargs = {'station': '1'}
+            cache(elog=HutchELog.from_conf(hutch.upper(), **kwargs))
```

Comment:
What is this?

Reply:
- moved elog earlier in the creation process for organization sake
- skip elog if no hutch e.g. a raw `hutch-python` call

Reply:
Cool.